### ISSUE TITLE
Supervised or Unsupervised ?

### ISSUE THREAD

**Issue by apexx77** [facebookresearch/adaptive_teacher#18]
### Need clarification whether Adaptive Teacher is unsupervised or not.
https://github.com/facebookresearch/adaptive_teacher/blob/cba3c59cadfc9f1a3a676a82bf63d76579ab552b/adapteacher/engine/trainer.py#L646-L647

Here are you using the labels of Target dataset?

**Comment by yujheli** [comment]
@apexx77 This line of code is for keeping the weights as default (1) for supervised loss in the source domain. It would be great to have you read through the codes at: https://github.com/facebookresearch/adaptive_teacher/blob/main/adapteacher/engine/trainer.py#L495

Then you will better understand how we process the labels for the target (removing labels in target and replacing them with pseudo labels) and train the model with source labels. Thanks!

**Comment by apexx77** [comment]
### Scenario -1
   - Created the same dataset directory as mentioned in the readme for CS and Foggy CS
   - Started training Adaptive Teacher

### Scenario-2
   -  Created Coco annotations for CS and Foggy CS.
   - Registered the datasets in `adaptive_teacher/adapteacher/data/datasets/builtin.py`
   - Started training Adaptive Teacher 
### Expected behaviour
   - Getting almost same results in both the scenarios
### Actual behaviour
   - Getting the expected results in Scenario-1 but getting very low results in Scenario-2
   - In the initial supervised burn up steps, Scenario-2 has very low AP as compared to Scenario-1

@yujheli

**Comment by yujheli** [comment]
Could you check if you COCO annotations are well-created? Also check if you use the correct register function for detectron2 such as setup correct meta data or so.

**Comment by apexx77** [comment]
Yes, I did bbox visualisation and trained supervised models in detectron2 as well. But getting low AP after supervised iterations in Adaptive Teacher
